src/main/java/io/vertx/ext/mongo/impl/MongoClientImpl.java

```diff
@@ -940,11 +940,11 @@ public MongoClient runCommand(String commandName, JsonObject command, Handler<As
 
   @Override
   public MongoClient distinct(String collection, String fieldName, String resultClassname, Handler<AsyncResult<JsonArray>> resultHandler) {
-    return distinct(collection, fieldName, resultClassname, resultHandler, null);
+    return distinct(collection, fieldName, resultClassname, null, resultHandler);
   }
 
   @Override
-  public MongoClient distinct(String collection, String fieldName, String resultClassname, Handler<AsyncResult<JsonArray>> resultHandler, DistinctOptions distinctOptions) {
+  public MongoClient distinct(String collection, String fieldName, String resultClassname, DistinctOptions distinctOptions, Handler<AsyncResult<JsonArray>> resultHandler) {
```

Comment:
This is a legitimate breaking change. We must document it in the breaking changes wiki page after merging.

Reply:
Shall I document it in the wiki and/or move it to another PR (like above's issue with `collation` vs `collationOptions`)?

Reply:
I'm not sure you can update the wiki without being a committer. Let's talk about it after the PR is merged